sjb/config/test_cases/test_branch_origin_extended_conformance_install_containerized.yml

```diff
@@ -70,15 +71,19 @@ extensions:
                          -e containerized=true      \
                          -e deployment_type=origin  \
                          /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-node/network_manager.yml
+        # add extra tag to docker images in a form of v#.#.#.#
+        for image in $(docker images --format "{{.Repository}}:{{.Tag}}:" | grep ":$( cat ./ORIGIN_COMMIT ):" | cut -d':' -f1 | sort -u); do
+                docker tag ${image}:$( cat ./ORIGIN_COMMIT ) ${image}:$( cat ./ORIGIN_TAG )
```

Comment:
We didn't like this approach as if it failed, it would silently fail and fall back to pulling the image from a remote registry instead of testing the code we are trying to test.